src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  width: 200px;
+  padding: 29px 14px 14px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $standard-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 45px;
+  }
+
+  &__name {
+    margin: 0 0 6px;
+    font-weight: $fw-semibold;
+    font-size: $fs-md;
+    line-height: 15px;
+  }
+
+  &__code {
+    margin: 0 0 15px;
+    align-self: flex-start;
+    font-size: $fs-sm;
+    line-height: 13px;
+  }
+
+  &__rating {
+    width: 100%;
+    margin: 0 0 20px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__stars .stars__star:nth-child(-n+4) {
+    background: url("../images/star-active.svg") no-repeat 50%;
+  }
+
+  &__feedback {
+    margin: 0;
+    display: flex;
+    align-items: center;
+    font-size: $fs-sm;
+    line-height: 13px;
+  }
+
+  &__price {
+    width: 100%;
+    margin-bottom: 20px;
```

Comment:
I'm not quite sure, but maybe we should use paddings instead of margins inside element (BEM)

Reply:
I think it's perfectly fine to use margins inside block (it's not the same as using margins outside the block - this is not good)